src/mscorlib/src/System/ReadOnlySpan.cs

```diff
@@ -328,7 +328,7 @@ public T[] ToArray()
         public static ReadOnlySpan<T> Empty => default(ReadOnlySpan<T>);
     }
 
-    public static class ReadOnlySpanExtensions
+    public static class ReadOnlySpanExtensionMethods
```

Comment:
@jkotas, can this discussion/change be part of a separate PR. I want to get fast span enabled.